### ISSUE TITLE
Full setup 

### ISSUE THREAD

**Issue by benoitgaudou** [CityScope/CSL_Hanoi#5]
Hi all, 
I just reviewed some of the discussions by email.
I would like to launch the discussion about the full hardware setup needed for this project.

If I am not wrong we need:
- 1 computer/laptop to run the simulation
- a way to project from the top, which includes: 1 projector, something to carry the projector. As an example, below is a picture of the HKA setup.
- lego blocks 
- the table: @agrignard do you have picture of yours. It was quite easy to built the box in wood for HKA (in small shops in Hanoi). For this project, do we need something with a higher depth (in order that the camera has enough space to see everything)? we need to have on the top something transparent I guess :-), but it is all the table or just a part? 
- a camera to put in the box ? (which kinf of camera ?). Is it a USB camera we can plug in the laptop used for the simulation ? 

Did I miss something ? 

![P1223578](https://user-images.githubusercontent.com/1555274/62099088-2d488f00-b2b7-11e9-8fd9-d5354a033aa4.jpg)

Cheers

**Comment by agrignard** [comment]
For the projector, it usually looks better if the projector are directly mounted on the sailing so people can freely move around the table, if not we usually use 80/20 frames (8020.net)

For the table itself indeed it's a bit more complex than just a frame if we want to have a scanning part.

Here is one of the solutions that we use in different project:

https://github.com/CityScope/cityscope.github.io/blob/master/CS_Hardware/CS%20TABLE_KP_15/DRAWING-ASSEMBLY.pdf

**Comment by khanhnt** [comment]
Is it the acrylic transparent, @agrignard ? We will put the grid on the plate, and the camera in the box to capture the QR code, and send it then to the server.

**Comment by agrignard** [comment]
Yes you need a transparent acrylic to be able to see from the bottom the lego Qr Code

**Comment by agrignard** [comment]
Here is a full picture showing the actual setup to replicate in Hanoi
<img width="1022" alt="CityScope_Hanoi_setup" src="https://user-images.githubusercontent.com/3928502/67799101-1a329000-fa5b-11e9-8dc0-5a70b57e087b.png">

**Comment by agrignard** [comment]
For the projector, this is a list of compatible ones

https://www.projectorcentral.com/projectors.cfm?g=1&hide=0&st=1&exp1=1&r=13&br=4000-7000&c=&w=&db=&ar=&zr=&wt=&ltg=&ll=&wr=&dt=3.0.0&mfg=&p=&t=&pjl=0&pjw=0&pjh=0&td=&is=&i=d&tr=&tr2=&exp5=1&oop=2&sort=pop&sz=15#top

**Comment by agrignard** [comment]
I close this issue to focus on more specific one as it seems that a big part of the hardware deployment is on a good track
